sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -416,7 +442,38 @@ private synchronized boolean instanceConfigure(Application application, String a
 
         /* Store state. */
         mApplication = application;
-        mAppSecret = appSecret;
+
+        /* A null secret is still valid since transmission target token can be set later. */
```

Comment:
How does the set "later" flow works? I have trouble figuring out just by looking at the PR.